content/push/index.textile

```diff
@@ -15,7 +15,7 @@ redirect_from:
   - /general/smart-notifications
 ---
 
-Push notifications notify user devices whether or not an application is open and running. They deliver information, such as app updates, social media alerts, or promotional offers, directly to the user's screen. Ably sends push notifications to devices using Google's Firebase Cloud Messaging service ("FCM":https://firebase.google.com/docs/cloud-messaging) and Apple's Push Notification Service ("APNs":https://developer.apple.com/notifications/). Push notifications don't require a device to stay connected to Ably. Instead, a device's operating system  maintains its own battery-effficient transport to receive notifications.
+Push notifications notify user devices whether or not an application is open and running. They deliver information, such as app updates, social media alerts, or promotional offers, directly to the user's screen. Ably sends push notifications to devices using Google's Firebase Cloud Messaging service ("FCM":https://firebase.google.com/docs/cloud-messaging), Apple's Push Notification Service ("APNs":https://developer.apple.com/notifications/), and the "Web Push API":https://developer.mozilla.org/en-US/docs/Web/API/Push_API. Push notifications don't require a device to stay connected to Ably. Instead, a device's operating system  maintains its own battery-effficient transport to receive notifications.
```

Comment:
Is "Instead, a device's operating system  maintains its own battery-effficient transport to receive notifications." still correct, in the context of browser?  In addition this text has a spelling mistake (fff) and an extra space after `system`

Reply:
fixed the typos, thanks.

as for correctness of that sentence, i've tweaked it slightly to clarify that web push subscriptions aren't handled by the OS